presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -577,4 +603,14 @@ private DataSetup mysqlCreateAndInsert(String tableNamePrefix)
     {
         return dataType("double precision", DoubleType.DOUBLE, Object::toString);
     }
+
+    private static DataType<byte[]> binaryDataType()
+    {
+        return dataType(
+                "varbinary(50)",
```

Comment:
You can make it `"varbinary(50)"` a parameter to this method.
Then you can easily add test coverage for  `"binary(50)"`